content/docs/io/overview.md

```diff
@@ -6,7 +6,109 @@ menu:
     parent: io
 ---
 
-This page has not been worked on yet. If you'd like to contribute visit the [doc-push]
-repo.
+Rust [`std::io`] module provides traits that synchronously access resources like
+files and sockets.  These `Read` and `Write` traits are implemented in other
+modules including [`std::fs`] and `std::net`.
 
-[doc-push]: https://github.com/tokio-rs/doc-push
+The [`tokio`] crate comes with a variation on those types, [`AsyncRead`] and
+[`AsyncWrite`].  The primary difference between [`AsyncRead`] and `std::io::Read`
+is the [`futures::Async`] return values.  The implementation returns an `Error`,
+`NotReady`, or `Ready` value.
+
+## Non Blocking
+
+The nature of a polling data reader/writer allows for other tasks to operate
```

Comment:
I would stick with "non-blocking" as the term (vs. polling).

"Non-blocking reading and writing allows for other tasks to operate concurrently" maybe?